CODEOWNERS

```diff
@@ -6,7 +6,6 @@
 
 /packages/gasket-cli/                       @indexzero, @kinetifex
```

Comment:
The codeowners file needs an overhaul. I think you may need to leave it alone in this PR and we're clean up later